packages/message/src/Granules.ts

```diff
@@ -8,18 +8,35 @@
  * const Granules = require('@cumulus/message/Granules');
  */
 
-import get from 'lodash/get';
 import { Message } from '@cumulus/types';
 
+interface MessageWithGranules extends Message.CumulusMessage {
+  payload: {
+    granules?: object[]
+  }
+}
+
 /**
- * Get granules from execution message.
+ * Get granules from a workflow message.
  *
- * @param {Message.CumulusMessage} message - An execution message
+ * @param {MessageWithGranules} message - A workflow message
  * @returns {Array<Object>|undefined} An array of granule objects, or
  *   undefined if `message.payload.granules` is not set
  *
  * @alias module:Granules
  */
 export const getMessageGranules = (
-  message: Message.CumulusMessage
-): unknown[] | undefined => get(message, 'payload.granules');
+  message: MessageWithGranules
+): unknown[] | undefined => message.payload?.granules;
+
+/**
+ * Determine whether workflow message has granules.
```

Comment:
Nit: as this is a double lookup, can we add a comment here noting we're talking about payload?. granules?

Reply:
`messageHasGranules` was removed, but added to `getMessageGranules`:

3f60ec0d700e653789b08cd3be3c913c2e287d90